salesagility/SuiteCRM
# Fix #9783 - Compose view quick search for email templates

## Description
See https://github.com/salesagility/SuiteCRM/issues/9783

Email templates doesn't activate quick search when typing in the field in the Emails module compose view

## Motivation and Context
Quick search feature is handy to have on this relate field so that opening the pop-up isn't required

## How To Test This
1. Go to the Emails module
2. Click on Compose
3. In the Email Template Field type "C"
4. A list of email templates beginning with C should be displayed
5. No email template options are displayed

## Types of changes


### Final checklist
